internal/action/actions.go

```diff
@@ -2064,22 +2064,25 @@ func (h *BufPane) MouseMultiCursor(e *tcell.EventMouse) bool {
 	return true
 }
 
-// SkipMultiCursor moves the current multiple cursor to the next available position
-func (h *BufPane) SkipMultiCursor() bool {
+// SkipMultiCursor moves the current multiple cursor to the next or previous available position, depending on the argument `forward`.
+func SkipMultiCursor(h *BufPane, forward bool) bool {
```

Comment:
Erg, this looks ugly. I didn't meant this helper function not to be a method of `BufPane`.

The most obvious way is to name it `skipMultiCursor()` i.e. non-capitalized, i.e. private (it doesn't need to be public, right?). I.e.:

```
func (h *BufPane) skipMultiCursor(forward bool) bool
```

Reply:
And the comment above it wouldn't be necessary then.

Reply:
I don't mind changing this, but why does it look ugly if this is not a method of ‘BufPane’?

Reply:
Because if all other functions here dealing with a `BufPane` are methods, why should this particular one be not a method? Just to let it have the same name as the `SkipMultiCursor` method?

Reply:
Okay, thanks for the explanation.